Packs/CarbonBlackDefense/ReleaseNotes/4_0_0.md

```diff
@@ -0,0 +1,31 @@
+****
+#### Integrations
+
+##### New: Carbon Black Endpoint Standard v3
+
+- Deprecated the **cbd-find-events** and **cbd-find-events-results** commands, replaced by **cbd-find-observation**.
```

Comment:
Deprecated the **cbd-find-events** and **cbd-find-events-results** commands, replaced with the **cbd-find-observation** command.